peter-lang-verlag.csl

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded">
+  <info>
+    <title>Peter Lang Verlag</title>
+    <title-short>Peter Lang</title-short>
+    <id>http://www.zotero.org/styles/peter-lang-verlag</id>
+    <link href="http://www.zotero.org/styles/peter-lang-verlag" rel="self"/>
+    <author>
```

Comment:
> if possible, add a rel="template" link for the template you based your style on 
It's not based on another style. Should I add an empty link because of that?
ِ>‌ and also a rel="documentation" link with a link to the documentation.
The publisher has supplied me directly with their requirements and have told me that they are not available online. Also their instructions are not entirely unambiguous so our editors have encouraged us to look for books by our publisher in the library and consult those in order to get quotations, etc. right. How do you usually handle cases like that?